ios/FluentUI/Controls/TouchForwardingView.swift

```diff
@@ -27,11 +27,33 @@ open class TouchForwardingView: UIView {
             let localPoint = convert(point, to: view)
             if view.point(inside: localPoint, with: event) {
                 if event?.type == .touches {
-                    onPassthroughViewTouches?(event)
+                    // If onPassthroughViewTouches is nil then pass the touch events to passthroughView view
+                    guard let onPassthroughViewTouches = onPassthroughViewTouches else {
+                        return true
+                    }
+                    onPassthroughViewTouches(event)
                 }
                 return false
             }
         }
         return super.point(inside: point, with: event)
     }
+
+    open override func hitTest(_ point: CGPoint, with event: UIEvent?) -> UIView? {
+        var hitTestView = super.hitTest(point, with: event)
+
+        // If onPassthroughViewTouches is non-nil then don't pass the touch events to passthroughView view
+        if onPassthroughViewTouches != nil {
```

Comment:
Can you use guard ... else here instead?

From the swift guidelines (https://github.com/microsoft/swift-guide/blob/gh-pages/EarlyReturn.md):
"It's recommended to use guard ... else statements and early exits on top of a function to validate parameters or other variables."

Reply:
also it should be more like let foo = fooo {} rather than nil check

Reply:
@harrieshin  @rdeassis we don't want to use onPassthroughViewTouches that's why used if check here.

I can use guard for early return something like this:
guard onPassthroughViewTouches == nil else {
        return hitTestView
}

Reply:
Yeah. I think that the fact that you don't need onPassthroughViewTouches in the logic of this method, it's okay to do the nil check, but still with the guard statement as you mentioned:

        guard onPassthroughViewTouches == nil else {
            return hitTestView
        }

Reply:
fixed!